drizzle-orm/src/pg-core/session.ts

```diff
@@ -29,6 +30,14 @@ export interface PgTransactionConfig {
 	isolationLevel?: 'read uncommitted' | 'read committed' | 'repeatable read' | 'serializable';
 	accessMode?: 'read only' | 'read write';
 	deferrable?: boolean;
+	rlsConfig?: {
```

Comment:
What about (suggestion):
```ts
set?: {
		configs?: {
			name: string;
			value: string;
			isLocal?: boolean;
		}[];
		role?: AnyPgRole;
	};
```
- `set` because postgres commands start with `set` or `set_`
- `configs` because of `set_config(...)`

And I think it not only related to rls.
We can imagine wanting to use this to set_config, which is later used in a PG function that we called in our transaction.

Like https://www.postgresql.org/docs/current/sql-syntax-calling-funcs.html

Reply:
Yes, `set_config(...)` is not just for policies. I like `set: { configs: ..., role: ... }` or `set_configs: ..., set_role: ...`.

https://www.postgresql.org/docs/16/functions-admin.html